docs/PROJECTS.md

```diff
@@ -16,6 +24,18 @@ Gunderson Dettmer and @JSv4 are working on OCF export functionality for Incorpor
 
 Shoobx and @regebro are working on PyOCF, a Python library to read and create Open Cap Table Format files. Check it out [here](https://github.com/Shoobx/pyocf).
 
+### OpenCap.co
+
+[OpenCap.co](https://opencap.co/) is both an open source project a company. They're building an [open source cap table application](https://github.com/opencapco/opencap.co) that is compatible with the OCF standard.
```

Comment:
```suggestion
[OpenCap.co](https://opencap.co/) is both an open source project and a company. They're building an [open source cap table application](https://github.com/opencapco/opencap.co) that "will follow" the OCF standard.
```

I don't feel we should claim compatibility on their behalf; "will follow" is what their README says.

Reply:
Pushed this change, but I left off the double quotes